dask/tests/test_delayed.py

```diff
@@ -1,10 +1,12 @@
 from collections import namedtuple
 from operator import add, setitem
+from functools import partial
 import pickle
 from random import random
 import types
 
-from toolz import identity, partial, merge
+from tlz import merge
+from toolz import identity
```

Comment:
Is identity available from tlz?

```suggestion
from tlz import merge, identity
```

Reply:
It's used in the test `test_name_consistent_across_instances`.  The identity function from toolz generates a different key from the one in cytoolz. 

If any function could be used, perhaps use a pure function from the standard library?